src/components/custom-field-input-number/__snapshots__/custom-field-input-number.test.jsx.snap

```diff
@@ -40,11 +34,11 @@ exports[`CustomFieldInputNumber has defaults 1`] = `
 
 exports[`CustomFieldInputNumber has defaults 2`] = `
 Object {
-  "dirty": false,
+  "dirty": true,
   "focus": [Function],
   "id": "test-input",
   "name": "field-id",
   "validity": ValidityState {},
-  "value": "",
+  "value": "NaN",
```

Comment:
Do we want this change? It kinda seems weird. What do you think?

Reply:
I think the value of the number custom field defaults to NaN when the value is empty, and that's because of its ref. I think that's fine? I don't know. What do you think?

Reply:
I think it's weird that the CustomFieldInputNumber used to emit an empty string and now it emits NaN. It's a breaking change tbh. 

And I think we are relying on it being an empty string in bigmaven for our custom fields app serialization -- I can't fully recall. We'll want to make sure the changes are compatible with usages in bigmaven  -- rogue is already using the custom field input number component as well.

Reply:
Also, something to get your gears in the right direction,

> I think the value of the number custom field defaults to NaN when the value is empty, and that's because of its ref.

What has changed? Wasn't it a ref before as well? If it's "because it is a ref" then wouldn't it have been NaN in `master` too?

Reply:
This is outdated--it now defaults to undefined instead.